### PR TITLE
Release notes for 1.6

### PR DESCRIPTION
**Pull request by FelisiaM**
# Pull Request Template

Which other branches should this be merged with (if any)?

### CODE REVIEW

File: reference/gcp-postgresql.html.md.erb [hunk 1]
```diff
@@ -109,11 +109,31 @@ whether a parameter is supported for both provision and update, or for provision
     <td><code>storage_gb</code></td>
     <td>number</td>
     <td>
-      Required unless defined in the plan. Size of storage volume for service instance. 10&ndash;4096&nbsp;GB.
+      Size of storage volume for service instance. 10&ndash;4096&nbsp;GB. To change this value, set the `disk_autoresize` to false.
```
Comment (anita-flegg) [inline reference/gcp-postgresql.html.md.erb:112]: Please change the markdown code ticks on disk_autoresize to the HTML <code> markup.
The markdown markup doesn't work inside an HTML table.

File: reference/gcp-mysql.html.md.erb [hunk 2]
```diff
@@ -92,13 +92,23 @@ provisioning a csb-google-mysql service:
     <td><code>false</code></td>
     <td>provision and update</td>
   </tr>
+  <tr>
+  <td><code>storage_gb</code></td>
+  <td>number</td>
+  <td>
+    Size of storage volume for service instance. To change this value, set the `disk_autoresize` to false.
```
Comment (anita-flegg) [inline reference/gcp-mysql.html.md.erb:99]: Please change the markdown code ticks on disk_autoresize to the HTML  markup.
The markdown markup doesn't work inside an HTML table.

**Comment by anita-flegg** [comment]
I have added 2 comments.

Please also check the index file to ensure that the table there is up-to-date.
Ajayan told me that changes to the release notes often necessitate changes to the index file, too.
Thanks!

**Comment by FelisiaM** [comment]
Hi @anita-flegg 

Thanks for the review! I made the changes already. 

Most of the time changes in the docs-csb-gcp/docs-csb-aws/docs-csb-azure (which all have the same layout) will require changes in the `release-notes`, `upgrading` and reference files. These docs refer to service brokers that offer services such as MySQL, Postgress and more. Most often our team is just enhancing existing services. 
Only when we add or remove a service we would need to touch the reference/index file to reflect that. We might have some removals in the near future but this PR is the most common type of PR these docs get.

Thanks!

**Pull request merged**